app/helpers/questions_helper.rb

```diff
@@ -43,8 +43,8 @@ def first_question
     questions_to_ask.first
   end
 
-  def first_question_seen?
-    session[:questions_to_ask].present?
+  def last_question_seen?
+    session[:nation].present?
```

Comment:
Can you add a note about why this check is being changed?
Also, `first_question_seen?` is still being stubbed [here](https://github.com/alphagov/govuk-coronavirus-find-support/blob/master/spec/requests/need_help_with_spec.rb#L4)

Reply:
:+1: